Application/src/tracker/python/visual_identification_network_torch.py

```
@@ -497,13 +567,41 @@ def v110(num_classes, channels, image_width, image_height):
+            
+            # Always keep BatchNorm2d; avoid GroupNorm.
+            # Move to device and prefer channels_last activations on MPS so compiled convs
+            # see consistent NHWC (channels_last) layout during forward/backward.
+            try:
+                if str(device).startswith('mps'):
+                    #model = replace_batchnorm2d_with_groupnorm(model)
+                    model = model.to(device, memory_format=torch.channels_last)
+                else:
+                    model = model.to(device)
+            except Exception:
+                pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)